daml-foundations/daml-ghc/daml-prim-src/GHC/Enum.daml

```diff
@@ -69,61 +66,59 @@ class  Bounded a  where
 -- ```
 --
 class  Enum a   where
-    -- | the successor of a value.  For numeric types, 'succ' adds 1.
+    -- | Returns the successor of the given value. For example, for
+    -- numeric types, `succ` adds 1.
+    --
+    -- If the type is also an instance of `Bounded`, `succ maxBound`
+    -- results in a runtime error.
     succ                : a -> a
-    -- | the predecessor of a value.  For numeric types, 'pred' subtracts 1.
+    -- | Returns the predecessor of the given value. For example, for
+    -- numeric types, `pred` subtracts 1.
+    --
+    -- If the type is also an instance of `Bounded`, `pred minBound`
+    -- results in a runtime error.
     pred                : a -> a
-    -- | Convert from an 'Int'.
+    -- | Convert a value from an `Int` to an `Enum` value: ie, returns
+    -- the item at the `Int` position from the `Enum`.
```

Comment:
I'd say: `toEnum i` returns the item at the `i`th position of (the instance of) `Enum`.